src/components/diary-card/diary-card.js

```diff
@@ -0,0 +1,52 @@
+import { Button, Card, CardActions, CardContent, makeStyles, Typography } from "@material-ui/core";
+import { useState } from "react";
+
+const useStyles = makeStyles({
+    card: {
+        minWidth: 275,
+        textAlign: "start",
+        backgroundColor: "#b3e9fe",
+        height: '100%',
+        display: 'flex',
+        flexDirection: 'column'
+    },
+    title: {
+        fontSize: 20
+    },
+    name: {
+        marginBottom: "10px"
+    },
+    button: {
+        fontWeight: 600
+    },
+    cardContent: {
+        flexGrow: 1,
+    }
+});
+
+function DiaryCard(props) {
+
+    const classes = useStyles();
+    const [isMore, setMore] = useState(false)
```

Comment:
Issue in naming: 
If getter is **x**, setter should be **setX**. So in here, setter should be `setIsMore`.

Reply:
noted!